init/caddy.service

```diff
@@ -1,30 +1,25 @@
-# This service file requires the following:
+# caddy.service
 #
-# 1) Group named caddy:
-#      $ groupadd --system caddy
+# For using Caddy with a config file.
 #
-# 2) User named caddy, with a writeable home folder:
-#      $ useradd --system \
-#           --gid caddy \
-#           --create-home \
-#           --home-dir /var/lib/caddy \
-#           --shell /usr/sbin/nologin \
-#           --comment "Caddy web server" \
-#           caddy
-#
-# 3) Caddyfile at /etc/caddy/Caddyfile that is
-#    readable by the caddy user
+# Make sure the ExecStart and ExecReload commands are correct
+# for your installation.
 #
+# WARNING: This service does not use the --resume flag, so if you
+# use the API to make changes, they will be overwritten by the
+# Caddyfile next time the service is restarted. If you intend to
+# use Caddy's API to configure it, add the --resume flag to the
+# `caddy run` command or use the caddy-api.service file instead.
 
 [Unit]
-Description=Caddy Web Server
+Description=Caddy
```

Comment:
Why are you dropping the `Web Server` part? It's a fairly useful part of the description...

Reply:
Comparison with other web server unit files:
* httpd.service: `The Apache HTTP Server`
* nginx.service: `The nginx HTTP and reverse proxy server`
* lighttpd.service: `Lightning Fast Webserver With Light System Requirements`

Reply:
I don't mind having it there, but in the future I want to get away from the idea that Caddy is _just_ a web server. (Even though that's what most people use it for currently.) We don't really want to be like other web servers. Even right now, it is also a certificate manager. And someday it will be a task scheduler or a dynamic DNS client or... any long running program, really! A caddy for your apps and services and things.

Anyway, that's the answer to your question about why I took it out. (Hopefully whoever installed it already knows it is a web server? And more, of course...)